sdk/typespec/typespec_client_core/src/http/format.rs

```diff
@@ -37,7 +37,8 @@ impl<D: DeserializeOwned> DeserializeWith<JsonFormat> for D {
     }
 }
 
-/// The default format used for deserialization.
+/// A [`Format`] that deserializes response bodies using JSON.
+/// This is the default format used for deserialization.
```

Comment:
Nit: per markdown rules, these are actually a single rendered line as the "title" - the one-liner that will show up in an API listing on docs.rs, for example. And maybe that's fine, but if you intended it as a separate line you'll need a blank line in between.